package.json

```diff
@@ -10,68 +9,65 @@
     "typecheck": "tsc --noEmit",
     "lint": "eslint --cache --ignore-path ./.gitignore --ext .js,.jsx,.ts,.tsx .",
     "lint:fix": "npm run lint -- --fix",
-    "e2e": "npm exec cypress install && npm exec grafana-e2e run",
-    "e2e:update": "npm exec cypress install && npm exec grafana-e2e run --update-screenshots",
-    "server": "docker-compose up --build",
+    "e2e": "playwright test",
+    "server": "docker compose up --build",
     "sign": "npx --yes @grafana/sign-plugin@latest"
   },
-  "author": "Cribl, Inc.",
+  "author": "Criblcloud",
```

Comment:
Wondering about the author change here?

Reply:
Good catch!  The way I did this was to create a brand new plugin from scratch using Grafana's NPX toolkit.  So a lot of this was just "copied over" the old one.  This is one of those cases.

Thanks for scrutinizing all of it.  I'll fix this.